pom.xml

```diff
@@ -72,8 +72,8 @@ under the License.
   </contributors>
   
   <properties>
-    <mavenVersion>3.0</mavenVersion>
     <javaVersion>7</javaVersion>
```

Comment:
why not 8?

Reply:
This is first 3.x release that should support even Maven 3.2.5... imho move to 8 in 3.1.0